src/util/helpers.coffee

```diff
@@ -15,18 +15,19 @@ module.exports = (context, parameters = {}, currency = null, translations = {})
       precision: 2
 
   helpers =
-    'url-params': () ->
-      (text, render) ->
-        url = (render text).trim()
-        if url.length > 0
-          params = []
-          for key of parameters
-            params.push "#{key}=#{parameters[key]}"
-          if params.length > 0
-            params = params.join "&"
-            glue = if url.match /\?/ then "&" else "?"
-            url = "#{url}#{glue}#{params}"
-        url
+      'url-params': () ->
+        (text, render) ->
+          querystring = render text
+          if querystring
+            params = []
+            for key, value of parameters
+              params.push "#{key}=#{value}"
+              if params.length != 0
+                params = params.join "&"
+            
+                glue = if querystring.match /\?/ then "&" else "?"
+              else querystring = "#{querystring}#{glue}#{params}"
```

Comment:
- Better `if params.length > 0` but it's OK, it will never be less than `0`.
- The `else` is wrong and shouldn't be there or the `glue` won't ever be used if there're params.

```coffee
if params.length > 0
  params = params.join "&"
  glue = if querystring.match /\?/ then "&" else "?"
  querystring = "#{querystring}#{glue}#{params}"
```